### DISCUSSION TITLE
Allow different challenge mode characters to interact with eachother (when it makes sense)

### DISCUSSION THREAD

**Discussion by clovaek** [chromiecraft/chromiecraft#5529] [Suggestions]
Looking at the current bloodthirsty + hardcore characters, I don't see a reason why they shouldn't be allowed to interact with hardcore characters.

The two challenge modes in the example aren't different in difficulty or character power. But due to the lesser popularity of bloodthirsty + hardcore there is hardly a community to play with, making trading or dungeons at current level non-existent.

**Comment by trinitarius** [comment]
I agree with this. Bloodthirsty shouldn't have a grouping restriction. It doesn't really give you an unreasonable advantage if compared with no grouping restrictions on Ironman or Hardcore challenge.

So following grouping should be possible:
HC/IM/BT + HC/IM
HC/BT + HC
IM/BT + IM
BT only + non-challenge players.

**Reply by Apathyxia** [reply]
In my opinion, even BT only should not be able to group up with non-challenge players. This would just make boosting very easy. And now if there's added a restriction that you wouldn't get xp if a non-challenge player damages the same mob, this grouping loses its point.

**Reply by Leckman98** [reply]
HC/IM/BT + HC/IM, HC/BT + HC, IM/BT + IM groupings are absolutely fair. That leaves BT-only in a weird spot and I agree with what Apathyxia said. If you ask me, the solution is simple, why would anyone play BT only? Just make it so BT can be only picked as a 2nd or 3rd challenge, and not as a 1st challenge.

**Comment by Dudududy** [comment]
I think it could make sense to allow some grouping, because as mentioned some challenge combinations have low popularity (for now).
However, non-challenge players should never be allowed to group up.

I would even put on more restrictions on grouping up - like level restrictions to prevent boosting, which is already happening and kinda defeats the whole challenge purpose.

**Comment by Peti321** [comment]
I think that it splits the playerbase in too many ways. Some challenges like BT+IM can hardly find other players to interact with.
I didnt want to make a new suggestion so I write it here:
In the game the challenge players are often categorized by 1/3 or 2/3 and 3/3.
But even then I have to check it, whether we can trade and group up, because BT+IM cant group up with HC+IM.
My suggestion is to make the number of challenges on a character to matter in the interactions.
So any 1/3 character can interact with any 1/3 characters. 
All 2/3 with all 2/3 and the 3/3 are alone.
In this way there are only 4 "groups" of players. 0 challenge, 1 challenge, 2 challenge, 3 challenge.
(Currently there are 8 "groups" )
